iosxr_iso2vbox.py

```diff
@@ -69,29 +69,54 @@
 import argparse
 from argparse import RawDescriptionHelpFormatter
 import re
+import logging
 
 # Telnet ports used to access IOS XR via socat
 console_port = 65000
 aux_port = 65001
 
 
-def run(command_string, debug=False):
+def run_process_quietly(cmd_string, debug=False):
     '''
-    Execute a CLI command string and return the result.
+    Execute a CLI command in the 'background' as process.
+    '''
+    argv = shlex.split(cmd_string)
+    if debug is True:
+        print ('argv: %s' % argv)
+    with open(os.devnull, 'w') as fp:
+        subprocess.Popen((argv), stdout=fp)
+    time.sleep(2)
 
-    Note this does not handle operators which need whitespace.
 
+def run_get_output(cmd_string, debug=False):
+    '''
+    Execute a CLI command string and return the output.
+    Note this does not handle operators which need whitespace.
     In those cases the raw commands will need to be used.
-
     E.g. passing grep 'a string separated by spaces'.
     '''
-    argv = command_string.split()
+    argv = cmd_string.split()
     if debug is True:
         print ('argv: %s' % argv)
```

Comment:
Why not:

```python
logger.debug('argv: %s', argv)
```

Seems strange that you're adding print statements as part of a PR about moving away from them. 😄